zuul.d/projects.yaml

```diff
@@ -463,12 +463,6 @@
     templates:
       - ansible-python-jobs
       - ansible-python3-jobs
-    check:
-      jobs:
-        - ansible-tox-py27
-    gate:
-      jobs:
-        - ansible-tox-py27
```

Comment:
do you want this?

Reply:
ugh, not sure why I keep picking on other projects